components/connect/components/search/view.coffee

```diff
@@ -15,8 +15,14 @@ module.exports = class ConnectSearchView extends Backbone.View
     input: ->
       @query arguments...
 
-  initialize: ({ @state, @search }) ->
+  initialize: ({ @state, @search, @eventBus }) ->
     @query = throttle @perform, 200
+    
+    @eventBus.on 'connection:added', @afterConnection
+
+  afterConnection: =>
+    @$el.focus()
+    @$el.select()
```

Comment:
`on` isn't automatically bound to the view (was following your pattern in other views) but if listenTo works then cool.